op/model.py

```diff
@@ -386,3 +502,17 @@ def pod_spec_set(self, spec, k8s_resources):
             self._run('pod-spec-set', *args)
         finally:
             shutil.rmtree(tmpdir)
+
+    def status_get(self, is_app):
+        """Get a status of a unit or an application.
+        app -- A boolean indicating whether the status should be retrieved for a unit or an application.
+        """
+        return self._run('status-get', '--include-data', f'--application={is_app}')
+
+    def status_set(self, status, message='', is_app=None):
+        """Set a status of a unit or an application.
+        app -- A boolean indicating whether the status should be set for a unit or an application.
+        """
+        if not isinstance(is_app, bool):
```

Comment:
It doesn't make sense to default to None but crash unless it's a boolean. It needs to default to False as well. Let's also make both this and the the _get force is_app to be a kwarg (with `(..., *, is_app=False)`) so we don't get code looking like get(True) anymore.

Then, have you double-checked that status-set indeed accepts a parameter that looks like "--application=True/False"?

Was there

Reply:
I'll make the change.

As for `--application`, yes, it does:

```
juju run --unit ceph-osd/leader 'status-set --application=True active ; echo $?'
0
```

Reply:
Forced `is_app` to be a `kwarg` here:
https://github.com/canonical/operator/pull/52/commits/319b89126ee803ae5ffc58c2c2a5174118663254